test/output_formatter_test.cc

```diff
@@ -125,12 +125,18 @@ TEST_F(OutputCollectorTest, DottedFormatter) {
                         "test/test_data/output_formatter.dotted.gold");
 }
 
+TEST_F(OutputCollectorTest, CsvFormatter) {
+  CsvOutputFormatterImpl formatter;
+  expectEqualToGoldFile((formatter.formatProto(collector_->toProto())).value(),
+                        "test/test_data/output_formatter.csv.gold");
```

Comment:
I assume that this output_formatter.csv.gold file should also be included in the change?

Reply:
You're right, I forgot about the .gold file... I'll add it right now.